Base/QTCLI/qSlicerCLIExecutableModuleFactory.h

```diff
@@ -21,7 +21,7 @@
 #ifndef __qSlicerCLIExecutableModuleFactory_h
 #define __qSlicerCLIExecutableModuleFactory_h
 
-// SlicerQT includes
+// SlicerQt includes
```

Comment:
Since we are modifying the files, I suggest to use `// Slicer includes`

Reply:
Specifically changing `// SlicerQt includes` to `// Slicer includes` across all files used?

Reply:
Indeed, since the files live in the `Slicer/Base/*` directories, I think simply using `Slicer` makes sense.

On the other hand, files living in `Libs/*` are referenced using `MRML includes, ...`